Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -15,11 +16,11 @@ final class PaymentViewController: UIViewController {
 
   private(set) var viewModel: PaymentViewModel
   private var notificationCenter = NotificationCenter.default
-    private let cardValidator: CardValidating
+  private var cardValidator: CardValidator
```

Comment:
I can't see why this needs to become a variable, from code review I can't see it being overriden in new code 🤔